script/setup/critools-version

```diff
@@ -1 +1 @@
-v1.27.0
+v1.27.0
```

Comment:
Please remove unexpected symbol. And would you please to update the cri-tools test with empty filter? It was a good test case.

Reply:
In the v1.27.0 version, cri-tools test is still an empty filter, I submit the pr again to the cri-tools project and modify it back.

Reply:
Thanks!